src/datetime/MonthsView.js

```diff
@@ -22,17 +23,13 @@ export default class MonthsView extends React.Component {
 		let year = this.props.viewDate.year();
```

Comment:
In some views the method to render the header is called `renderHeader` and in others `renderNavigation`. Why don't we agree on use one name for all? I'd prefer `renderNavigation`, header is too general and the `DayView` has a secondary header, so when we talk about the navigation we know what we are talking about.

I'd also call the new component `ViewNavigation` instead of just `Header`.

Reply:
Sounds great!

Reply:
Done!